src/components/ActionCard.tsx

```diff
@@ -72,6 +73,10 @@ const styles = StyleSheet.create({
     borderWidth: 1,
     borderStyle: 'dotted',
   },
+  cardName: {
+    fontFamily: Fonts.FiraSansCondensedMedium,
```

Comment:
this is the default, we don't have to manually assign fontFamily, they are globally declared.